pkg/kubectl/cmd/config/unset.go

```diff
@@ -35,18 +35,28 @@ type unsetOptions struct {
 	propertyName string
 }
 
-var unset_long = templates.LongDesc(`
+var (
+	unsetLong = templates.LongDesc(`
 	Unsets an individual value in a kubeconfig file
 
 	PROPERTY_NAME is a dot delimited name where each token represents either an attribute name or a map key.  Map keys may not contain dots.`)
 
+	unsetExample = templates.Examples(`
+		# Unset the current-context.
+		kubectl config unset current-context
+
+		# Unset namespace in foo context.
+		kubectl config unset contexts.foo.namespace`)
```

Comment:
Are you sure this will work?
Is `contexts` a list?

Reply:
```
{
            "name": "ac",
            "context": {
                "cluster": "minikube",
                "user": "minikube",
                "namespace": "sdf"
            }
        },
        {
            "name": "boo",
            "context": {
                "cluster": "",
                "user": ""
            }
        },
        {
            "name": "default/127-0-0-1:8443/system:admin",
            "context": {
                "cluster": "127-0-0-1:8443",
                "user": "system:admin/192-168-65-2:8443",
                "namespace": "default"
            }
        },
```
`foo` is contexts name, this will unset namespace in foo context, have unit test in unset
_test.go